### DISCUSSION TITLE
Very confused about config.toml.

### DISCUSSION THREAD

**Discussion by tomleary** [jpanther/congo#295] [Q&A] [answered]
The instructions tell me to delete the config.toml file that Hugo generated, but that breaks everything. The site works if I leave it in, but now it supercedes other files in themes/congo/config/_default. For example, none of the changes I make in languages.en.toml take effect. Only changes to the original config.toml take effect. For some reason, changes to params.toml seem fine.

What's going on? If it makes a difference, I used the manual installation from Github.

**Comment by jpanther** [comment, marked as answer]
Hugo only allows you to have one site configuration so if you choose to keep using the config file that Hugo generated then you cannot use the ones that come with the theme (and you'd have to bundle all these parameters into your single config file). Are you sure you're setting up your project correctly and **copying the files** in the `your_project/themes/congo/config/_default` directory to `your_project/config/_default`?

As long as you have that copy in your project, when you delete the root config.toml the other ones should take effect. I would encourage you to read the [installation docs](https://jpanther.github.io/congo/docs/installation/#set-up-theme-configuration-files) again and follow them exactly.

**Reply by tomleary** [reply]
So to be clear, I need to create the directory "config" in the main directory, create a subdirectory called "_default", then copy into it the .toml files from "theme/congo/config/_default", right?

I did that and now I have a different set of issues. The live server displays a blank field:
<img width="450" alt="Screen Shot 2022-09-10 at 7 17 19 PM" src="https://user-images.githubusercontent.com/82277734/189505062-5021ec1f-0932-41e2-8a51-e865d5b1a3c8.png">

And the terminal outputs this error:

```
WARN 2022/09/10 19:21:12 found no layout file for "HTML" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/09/10 19:21:12 found no layout file for "JSON" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
```

This is what my file tree currently looks like:
<img width="364" alt="Screen Shot 2022-09-10 at 7 18 01 PM" src="https://user-images.githubusercontent.com/82277734/189505114-55dabdb9-e1b9-4340-b6ed-d7e50dd66083.png">

**Reply by tomleary** [reply]
I think I got it now. I suppose that I was confused by the instructions because I assumed the config folder they were talking about was the one within the theme folder. Thanks.

**Reply by jpanther** [reply]
Yes, you need a config folder in your Hugo project folder. The one in the theme just provides sensible defaults and then you override them using your own preferences. As a general rule, you should never change the files within the themes directory.

**Reply by mih** [reply]
Just leaving a note that I ran into this (or a similar) issue. The key bit in the instructions for me was this

![image](https://github.com/jpanther/congo/assets/136479/748156f4-b267-410f-b5e7-f3a4374f81b5)

Specifically the "important note" about adding the critical line to `config.toml`.
